pages/common/gh-workflow.md

```diff
@@ -0,0 +1,36 @@
+# gh workflow
+
+> List, view, and run workflows in GitHub Actions.
+> More information: <https://cli.github.com/manual/gh_workflow>.
+
+- Interactively select a workflow to view latest jobs:
+
+`gh workflow view`
+
+- View a specific workflow in the default browser:
+
+`gh workflow view {{id|filename.yml}} --web`
```

Comment:
Shouldnt' this be a path?  But it's starting to look really ugly...😞
```suggestion
`gh workflow view {{id|path/to/file.yml}} --web`
```

Reply:
no, it's just the filename of a file in the `.github/workflows/` directory in the repo. I think they keeped the `.yml` extension to differentiate `filename.yml` vs `workflow_name`.

Reply:
Hmm, good to know. 👍🏻